src/components/manage/Blocks/Block/StyleWrapper.jsx

```diff
@@ -0,0 +1,16 @@
+import cx from 'classnames';
+import { Container } from 'semantic-ui-react';
+import { buildStyleClassNamesFromData } from '@plone/volto/helpers';
+
+const StyleWrapper = ({ children, data }) => {
+  const styles = buildStyleClassNamesFromData(data?.styles);
+
+  return (
+    <div className={cx(styles)}>
```

Comment:
We're missing "native integration", we don't want to wrap `<p>` in divs. Also, I don't think it's safe to use `<Container>` here, or we need, at least, a way to tell if we're already in a container or not. A "contextual container", something that provides a react context so that further down we know if we can use the container or not.

The style wrapper could be a primitive that could also be used for "other things", columns, sections of a block, etc.

Reply:
At a first glance, I wouldn't add a Container either way here. It's not the purpose of this component, am I right?

Reply:
Removed container.